src/install/kubernetes/helm/install-helm.mdx

```diff
@@ -0,0 +1,121 @@
+---
+componentType: default
+headingText: Install the Kubernetes integration
+---
+
+New Relic has a bunch of Helm charts for the different components, each with its own set of features for the platform. You can install these components separately, but we really recommend using the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. New Relic provides this chart, which acts as a wrapper or a meta-package for the individual charts.
+
+To install `nri-bundle` with Helm follow these steps:
+
+1. Make sure `kubectl` is installed and configured correctly to manage the cluster.
+
+2. Use this command to add the New Relic Helm charts repo:
+    ```shell
+    helm repo add newrelic https://helm-charts.newrelic.com
+    ```
+3. Create a file named `values-newrelic.yaml`, which will be used to define your configuration:
```

Comment:
I would add a link to the YAML we have in our repo. This is a more complete values file with more parameters - https://github.com/newrelic/helm-charts/blob/master/charts/nri-bundle/values.yaml